config/initializers/rev_manifest.rb

```diff
@@ -0,0 +1,5 @@
+rev_manifest_path = 'public/assets/rev-manifest.json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.